requirements/base.in

```diff
@@ -7,7 +7,7 @@ django-oauth-toolkit<1.0
 djangorestframework>=3.0,<3.7     # API tools
 django-model-utils        # Provides TimeStampedModel abstract base class
 edx-opaque-keys>=0.4.2    # Provides CourseKey and UsageKey
-edx-completion
+edx-completion<1.0
```

Comment:
edx-completion is using 0.x on solutions, because 1.0+ pulls in a different version of edx-drf-extensions, which changed imports around, and had an incompatibility with the aggregator that we will need to resolve before we can upgrade.